instana/collector/base.py

```diff
@@ -13,10 +13,7 @@
 from ..util import every, DictionaryOfStan
 
 
-if sys.version_info.major == 2:
-    import Queue as queue
-else:
-    import queue # pylint: disable=import-error
+import queue # pylint: disable=import-error
```

Comment:
Is the comment needed?

Reply:
Keeping the `pylint` comment as discussed.